### PR TITLE
[SMALLFIX][branch-1.5] Fix a bug that mounting UFS should connect to UFS only when replay=true

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
@gpang  PTAL

File: core/server/master/src/main/java/alluxio/underfs/MasterUfsManager.java [hunk 1]
```diff
@@ -24,17 +20,6 @@
 public final class MasterUfsManager extends AbstractUfsManager {
 
   /**
-   * Establishes the connection to the given UFS from master.
-   *
-   * @param ufs UFS instance
-   * @throws IOException if failed to create the UFS instance
-   */
-  public void connect(UnderFileSystem ufs) throws IOException {
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/underfs/MasterUfsManager.java]: master doesnt need to connect anymore?

Comment (apc999) [inline core/server/master/src/main/java/alluxio/underfs/MasterUfsManager.java]: as discussed,  connection still made right after ufs created and when replay==false

**Pull request merged**